core/forkid/forkid_test.go

```diff
@@ -32,8 +32,13 @@ import (
 func TestCreation(t *testing.T) {
 	mergeConfig := *params.MainnetChainConfig
 	mergeConfig.MergeNetsplitBlock = big.NewInt(18000000)
+
+	withdrawalConfig := *params.MainnetChainConfig
+	withdrawalConfig.MergeNetsplitBlock = big.NewInt(18000000)
+	withdrawalConfig.ShanghaiTime = big.NewInt(1668000000)
 	type testcase struct {
 		head uint64
+		time uint64
```

Comment:
Kind of an annoying diff, I could see this broken out into a new test just for testcases that also have time. But just a minor nit.